payments-core/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -393,7 +393,7 @@ constructor(
     override suspend fun getPaymentIntentResult(data: Intent) =
         paymentIntentFlowResultProcessor.processResult(
             PaymentFlowResult.Unvalidated.fromIntent(data)
-        )
+        ).getOrThrow()
```

Comment:
Keeps the current behavior for now. Ultimately, I’d like to also update the places where this is used.